packages/react/src/publication/filters.ts

```diff
@@ -0,0 +1,14 @@
+import { PublicationContentWarning, PublicationMainFocus } from '@lens-protocol/api-bindings';
+
+export type PublicationFilters = {
+  restrictPublicationMainFocusTo?: PublicationMainFocus[];
+  restrictPublicationTagsTo?: {
+    all?: string[];
+    oneOf?: string[];
+  };
+  showPublicationsWithContentWarnings?: {
+    oneOf?: PublicationContentWarning[];
+  };
```

Comment:
what is the default behaviour when this is not provided?

Reply:
It should not be included in the API request.

Reply:
nit: I believe as of now one can provide and empty object. Is this intentional?
```ts
showPublicationsWithContentWarnings: {}
```